src/Persistence_SQL.php

```diff
@@ -233,7 +233,7 @@ public function initField($q, $field)
      */
     public function initQueryFields($m, $q, $fields = null)
     {
-        if ($fields) {
+        if (is_array($fields)) {
```

Comment:
maybe `is_array($fields) && $fields)`

Reply:
`[]` should be the same as `false` - no fields